### PR TITLE
Try noarch build

### PR DESCRIPTION
**Pull request by ivaquero**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,13 +15,14 @@ source:
 build:
     number: 1004
     script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
    script: python -m pip install . -vv
```

Other options are now the default.

**Comment by hmaarrfk** [comment]
there were about 3 other items in the checklist that you conveniently deleted.

2/3 were useful.
- [ ] Bumping the build number.
- [x] Calling the bot to rerender the recipe for by including the following phrase:

@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,13 +15,14 @@ source:
 build:
     number: 1004
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
    number: 1005
```

Please bump the build number to differentiate the builds

File: recipe/meta.yaml [hunk 3]
```diff
@@ -15,13 +15,14 @@ source:
...
     host:
-        - python
+        - python >=3.7
         - setuptools
     run:
-        - python
+        - python >=3.7
 
 test:
     imports:
```
Comment (hmaarrfk) [inline recipe/meta.yaml:26]: The license below should be

```
license: BSD-3-Clause
```

With a dash and capital as the bot points out.

**Comment by ivaquero** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 4]
```diff
@@ -13,23 +13,24 @@ source:
   {{ hashtype }}: {{ hash }}
 
 build:
-    number: 1004
-    script: python setup.py install --single-version-externally-managed --record record.txt
+    number: 1005
+    script: python -m pip install . -vv
```
Comment (hmaarrfk) [inline recipe/meta.yaml]:
```suggestion
    script: {{ PYTHON }} -m pip install . -vv
```

Comment (hmaarrfk) [reply recipe/meta.yaml]: I'm not too sure what this would do. They should be equivalent, in theory.

**Comment by hmaarrfk** [comment]
I think the solution is to add pip as a dependency.

**Comment by ivaquero** [comment]
This is a wonderful tutorial for me.

**Comment by hmaarrfk** [comment]
Glad it helped!

**Comment by hmaarrfk** [comment]
I think you can ask to be a maintainer at a different time. You do alot to help simply by making PRs without being an active maintainer. 

Feel free to ask the original maintainers to join in, but I would rather give them a chance to reply before imposing (by clicking merge myself)

**Pull request merged**